src/Compatibility/Core/src/iOS/Renderers/EntryRenderer.cs

```diff
@@ -375,6 +375,7 @@ void UpdateText()
 				Control.Text = text;
 		}
 
+		[PortHandler]
```

Comment:
We would still need to port code related to the `Placeholder`, could you  add a comment here?

`[PortHandler ("Partially ported ...")]`

Reply:
Done! :)

Reply:
Maybe that needs to be done in this PR? The `Placeholder` property is already in the new handlers, so I think for this feature to be implemented completely, it needs to update the placeholder as well. Unless there is some specific reason why it can't be done now?

Reply:
From my side it makes sense also, I can do it in this PR :)

Reply:
@mattleibow since this was merged by hartez I will finish the Placeholder part in another PR :)